RadixWallet/Clients/CloudBackupClient/CloudBackupClient+Interface.swift

```diff
@@ -36,12 +39,13 @@ public struct CloudBackupClient: DependencyKey, Sendable {
 }
 
 extension CloudBackupClient {
+	public typealias StartAutomaticBackups = @Sendable () async throws -> Void
 	public typealias LoadDeviceID = @Sendable () async -> UUID?
 	public typealias MigrateProfilesFromKeychain = @Sendable () async throws -> [CKRecord]
-	public typealias DeleteProfileBackup = @Sendable (UUID) async throws -> Void
+	public typealias DeleteProfileBackup = @Sendable (ProfileID) async throws -> Void
 	public typealias CheckAccountStatus = @Sendable () async throws -> CKAccountStatus
-	public typealias LastBackup = @Sendable (UUID) -> AnyAsyncSequence<BackupResult?>
-	public typealias LoadProfile = @Sendable (UUID) async throws -> Profile?
+	public typealias LastBackup = @Sendable (ProfileID) -> AnyAsyncSequence<BackupResult?>
+	public typealias LoadProfile = @Sendable (ProfileID) async throws -> Profile?
 	public typealias LoadAllProfiles = @Sendable () async throws -> [Profile]
```

Comment:
We should return `containsP2PLinks` for every profile (i.e. `[(Profile, Bool)]`).
Using profile data [here](https://github.com/radixdlt/babylon-wallet-ios/pull/1122/files#diff-c4ae62b3f1ebe8cd50673c681cd202bc3b22760a37e90016f7781f9fd94acdc8R60) you can make this check:
```swift
Profile.checkIfProfileJsonContainsLegacyP2PLinks(contents: data)
```